assets/i18n/active.de.json

```diff
@@ -157,44 +158,44 @@
     "other": "Unbekannte Umfrageeinstellung: {{.Setting}}"
   },
   "poll.newPoll.votesettings.invalidSetting": {
-    "hash": "sha1-a0649f6d7040b480c543e843e12cd14de97ef6d6",
-    "other": "Die Anzahl der Stimmen musss eine positive Zahl kleiner gleich der Anzahl der Antwortoptionen sein. Gewählt wurde \"{{.MaxVotes}}\", aber die Anzahl der Antwortoptionen ist \"{{.Options}}\"."
+    "hash": "sha1-bf96d6b01204be0f86727d2253d235272e177ded",
+    "other": "Die Anzahl der Stimmen muss eine positive Zahl und kleiner oder gleich der Anzahl der Antworten sein. Du hast \"{{.MaxVotes}}\" angegeben, aber die Anzahl der Optionen ist \"{{.Options}}\".",
```

Comment:
Please remove a comma in the end of this line.

Reply:
Fixed